src/lib/api/posts/posts.api.ts

```diff
@@ -25,4 +32,25 @@ export class PostsApi {
   unlike(postId: string) {
     return this.client.delete(`/posts/${postId}/likes`);
   }
+
+  getRepliesByPostId(postId: string) {
+    return this.client.get<PaginatedPost>(`/posts/${postId}/replies`);
+  }
+
+  createReplyForPost(
+    postId: string,
+    text: string,
+    mediaBlob?: Blob,
+    fileName?: string,
+  ) {
+    const form = new FormData();
+
+    form.append("text", text);
+
+    if (mediaBlob) {
+      form.append("media", mediaBlob, fileName);
+    }
+
```

Comment:
Should we make a small private helper for this? Since its used in createReplyForPost and in create()